cocotb/triggers.py

```diff
@@ -161,11 +161,36 @@ def unprime(self):
 class Timer(GPITrigger):
     """Fires after the specified simulation time period has elapsed."""
     def __init__(self, time_ps, units=None):
+        """
+        .. Constructor. This RST comment works around sphinx-doc/sphinx#6885
+
+        Args:
+           time_ps: The time value. This can also be a 
+               :class:`fractions.Fraction` or a :class:`decimal.Decimal` instance.
```

Comment:
I'd move this sentence to the end

Reply:
I have since re-arranged that part a bit, please take another look.